fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java

```diff
@@ -27,28 +27,45 @@
 import com.starrocks.common.Config;
 import com.starrocks.common.FeConstants;
 import com.starrocks.common.util.NetUtils;
+import com.starrocks.persist.Storage;
 import org.apache.commons.net.util.SubnetUtils;
 import org.apache.commons.validator.routines.InetAddressValidator;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.net.Inet4Address;
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Properties;
 
 public class FrontendOptions {
     
+    public enum StartFeSpecifiedHostType {
+        FQDN,
+        IP,
+        NOT_SPECIFIED,
+    }
+
     private static final Logger LOG = LogManager.getLogger(FrontendOptions.class);
 
     private static final String PRIORITY_CIDR_SEPARATOR = ";";
+
+    public static final String HOST_TYPE_IP = "IP";
+    public static final String HOST_TYPE_FQDN = "FQDN";
```

Comment:
You have defined the StartFeSpecifiedHostType, there is no need to define the String values. You can use StartFeSpecifiedHostType.of() to get the specified enum from string.